review.md

```diff
@@ -20,7 +20,7 @@ This study investigates the feasibility of deploying Deep-Q-based deep reinforce
 2) Add some papers published recently to the literature review. Also, more papers are needed to be reviewed.
 3) Some suggestions to improve the literature review section.
 4) Clarify and explain your work contributions and the research gap in an appropriate section such as at the end of the introduction.
```

Comment:
> 4) Clarify and explain your work contributions and the research gap in an appropriate section such as at the end of the introduction.

We should definitely discuss this (+ over the last year the "research gap" might have changed :confused:)